pyvista/plotting/themes.py

```diff
@@ -2709,6 +2711,29 @@ def before_close_callback(
     ):  # numpydoc ignore=GL08
         self._before_close_callback = value
 
+    @property
+    def allow_empty_mesh(self) -> bool:  # numpydoc ignore=RT01
+        """Return or set whether to allow plotting empty meshes.
+
+        Examples
+        --------
+        Enable plotting of empty meshes.
+
+        >>> import pyvista as pv
+        >>> pv.global_theme.allow_empty_mesh = True
+
+        Now add an empty mesh to a plotter
+
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_mesh(pv.PolyData())
+        >>> pl.show()  # doctest: +SKIP
+        """
```

Comment:
Pyvista standard is to have an empty line at the end.

```suggestion

        """
```

Reply:
Thanks! I always forget which we do